packages/api-page-builder-so-ddb-es/src/definitions/blockEntity.ts

```diff
@@ -0,0 +1,58 @@
+import { Entity, Table } from "dynamodb-toolbox";
+import { Attributes } from "~/types";
+
+interface Params {
+    table: Table;
+    entityName: string;
+    attributes: Attributes;
+}
+
+export const createBlockEntity = (params: Params): Entity<any> => {
+    const { entityName, attributes, table } = params;
+    return new Entity({
+        name: entityName,
+        table,
+        attributes: {
+            PK: {
+                partitionKey: true
+            },
+            SK: {
+                sortKey: true
+            },
+            TYPE: {
+                type: "string"
+            },
+            id: {
+                type: "string"
+            },
+            name: {
+                type: "string"
+            },
+            type: {
```

Comment:
What is `type` ? Can you remind me?

Reply:
I took a chance to investigate this matter:
https://github.com/webiny/webiny-js/blob/809fcc5f4a4795d41e69d7fd1b07541e55408046/packages/app-page-builder/src/admin/views/Pages/Editor.tsx#L69
Look like this property is used by Page Builder Editor (Sorry, I cannot provide more deep details)
Possible values are "element" and "block" (but not same block as new entity I have created).

As new "Block" entity is mostly a "copy" of "pageElement" (as per requirements) and will be used in page builder too - same fields are present

Reply:
Yeah, but if we just look at the `type` field, it doesn't serve any purpose. That's why I'd just remove it.

We'll see how we'll deal with this on the frontend later.